resources/views/layouts/navbar.blade.php

```diff
@@ -64,6 +64,14 @@
                 $newItem['isCustom'] = count($itemsInCustom) > 0;
                 $menuItems[] = $newItem;
             }
+
+            /*This code is temporary until find a generic solution to implement new packages
+            with its own Menu into the Top Navbar
+            */
+            if (Request::path() === 'package-analytics-reporting'){
```

Comment:
Please review if it is possible to remove the hardcoded `package-analytics-reporting`